### ISSUE TITLE
Request to update the pre-trained Inception-BN model

### ISSUE THREAD

**Issue by dafukua** [apache/incubator-mxnet#2155]
Previously, I was trying to do fine-tuning based on [the pre-trained Inception-BN model](https://github.com/dmlc/mxnet-model-gallery/blob/master/imagenet-1k-inception-bn.md). And I encountered an issue that the fine-tuned model produced almost the same prediction scores for every test sample.

After discussing with a friend, I learned that the above issue can be fixed by properly passing the parameter `fix_gamma=False` in`mx.symbol.BatchNorm` (for Inception-BN, one needs to modify this [line](https://github.com/dmlc/mxnet/blob/master/example/image-classification/symbol_inception-bn.py#L18) accordingly in the symbol file).

The above issue seems to be caused by some legacy issue that the pre-trained Inception-BN model was trained by using an old version of MXNet. If so, can we expect to have an updated pre-trained Inception-BN model soon? Or at least document it for fine-tuning somewhere?

Thanks.

**Comment by tqchen** [comment]
Let us first make the fix by adding fix_gamma=False as you suggested, so things are consistent. Can you open a PR on this? Thanks!

**Comment by EasonD3** [comment]
@tqchen , thanks for the reply. Since this issue only occurs in Inception-BN during fine-tuning and some other models are also using `BatchNorm` with the default `fix_gamma=True`, changing the Inception-BN symbol file (i.e., [symbol_inception-bn.py](https://github.com/dmlc/mxnet/blob/master/example/image-classification/symbol_inception-bn.py)) may not be a good practice?
